.devcontainer/Dockerfile

```diff
@@ -131,193 +159,223 @@ RUN ["/bin/bash", "-c", "if [[ -v GIT_PROXY ]]; then git config --global http.pr
 #  see https://gist.github.com/electronjoe/a899e4bfbc2904cb353444386296c38e
 # Note the CFLAGS define below due to glibc deprecation of critical flag,
 #  see https://github.com/rdslw/openwrt/blob/e5d47f32131849a69a9267de51a30d6be1f0d0ac/tools/bison/patches/110-glibc-change-work-around.patch
-RUN wget --progress=dot:giga https://ftp.gnu.org/gnu/nettle/nettle-2.5.tar.gz && \
-    tar -xf nettle-2.5.tar.gz && \
-    cd nettle-2.5 && \
-    mkdir build && \
-    cd build/ && \
-    ../configure --disable-openssl --enable-shared --libdir=/usr/local/lib && \
-    make -j"$(nproc)" && \
-    make install && \
-    ldconfig -v && \
-    cd / && \
-    wget --progress=dot:giga https://www.gnupg.org/ftp/gcrypt/gnutls/v3.1/gnutls-3.1.23.tar.xz && \
-    tar xf gnutls-3.1.23.tar.xz && \
-    cd gnutls-3.1.23 && \
-    mkdir build && cd build && \
-    CFLAGS=-D_IO_ftrylockfile ../configure --with-libnettle-prefix=/usr/local && \
-    make -j"$(nproc)" && \
-    make install && \
-    ldconfig -v && \
-    cd / && \
-    rm -rf nettle* && \
-    rm -rf gnutls*
+RUN wget --progress=dot:giga https://ftp.gnu.org/gnu/nettle/nettle-2.5.tar.gz \
+    && tar -xf nettle-2.5.tar.gz \
+    && cd nettle-2.5 \
+    && mkdir build \
+    && cd build/ \
+    && ../configure --disable-openssl --enable-shared --libdir=/usr/local/lib \
+    && make -j"$(nproc)" \
+    && make install \
+    && ldconfig -v \
+    && cd / \
```

Comment:
There are two separate packages combined in one RUN here. I would see the structure to be more modular, if we could separate this in nettle and gnutls with to RUN statements.
The cleanup would than need to be moved to the appropriate section, of course.
You might need to check if `--with-libnettle-prefix=/usr/local` is a blocker for that, however, `--libdir=/usr/local/lib` suggests, that the first install only needs to complete successfully.